packages/web3-constants/evm/trader.json

```diff
@@ -1152,6 +1260,192 @@
         "Fantom": "",
         "Aurora": "",
         "Aurora_Testnet": "",
-        "Conflux": ""
+        "Conflux": "",
+        "Harmony": "",
+        "Harmony_Test": ""
+    },
+    "VENONSWAP_ROUTER_ADDRESS": {
+        "Mainnet": "",
+        "Ropsten": "",
+        "Rinkeby": "",
+        "Kovan": "",
+        "Gorli": "",
+        "BSC": "",
+        "BSCT": "",
+        "Matic": "",
+        "Mumbai": "",
+        "Arbitrum": "",
+        "Arbitrum_Rinkeby": "",
+        "xDai": "",
+        "Avalanche": "",
+        "Avalanche_Fuji": "",
+        "Celo": "",
+        "Fantom": "",
+        "Aurora": "",
+        "Aurora_Testnet": "",
+        "Conflux": "",
+        "Harmony": "0xf012702a5f0e54015362cBCA26a26fc90AA832a3",
+        "Harmony_Test": "0x8e9A3cE409B13ef459fE4448aE97a79d6Ecd8b4b"
+    },
+    "VENONSWAP_FACTORY_ADDRESS": {
+        "Mainnet": "",
+        "Ropsten": "",
+        "Rinkeby": "",
+        "Kovan": "",
+        "Gorli": "",
+        "BSC": "",
+        "BSCT": "",
+        "Matic": "",
+        "Mumbai": "",
+        "Arbitrum": "",
+        "Arbitrum_Rinkeby": "",
+        "xDai": "",
+        "Avalanche": "",
+        "Avalanche_Fuji": "",
+        "Celo": "",
+        "Fantom": "",
+        "Aurora": "",
+        "Aurora_Testnet": "",
+        "Conflux": "",
+        "Harmony": "0x7D02c116b98d0965ba7B642ace0183ad8b8D2196",
+        "Harmony_Test": "0x066bafafC84607a956967F6f050bD14A8347952F"
+    },
```

Comment:
Could you proof router address and factory address?

Reply:
YesI ref: https://github.com/harmony-one/docs-home/blob/edad98a13ab671f1a3d846ff8c87175c49d96280/developers/list-of-deployed-contracts.md?plain=1#L57

Reply:
@developerfred thanks! That's helpful.